public/js/monitor.js

```diff
@@ -68,8 +69,4 @@ if(document.querySelector(".email-scan")){
   document.querySelector(".email-to-hash").addEventListener("input", enableBtnIfEmailValid);
 }
 
-$(document).foundation();
```

Comment:
If we're now going jQuery free, I think you can remove the jquery env from the .eslintrc config file:

https://github.com/mozilla/blurts-server/blob/349e39226e78f22d8761c5b6ae5e1a42ae11bb95/.eslintrc.js#L5

Reply:
Oh, plus maybe nuke the [public/js/vendor](https://github.com/mozilla/blurts-server/tree/master/public/js/vendor) directory since we now have orphaned files.

Reply:
PR #176 submitted.